tools/travis-upload-to-gdrive.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+source tools/travis-helpers.sh
+
+set -euo pipefail
+
+echo "Uploading test results to google drive"
+export GOPATH="/tmp/go"
+go get github.com/prasmussen/gdrive
```

Comment:
can we download it using a link from here?
https://github.com/prasmussen/gdrive#downloads

Adding extra apt packages (in this case golang) can slowdown CI process.
And compiled version should not require gdrive.